doc/tutorials/using_ubuntu_as_sos.rst

```diff
@@ -0,0 +1,262 @@
+         ls /mnt/EFI/ubuntu
+
+      You should see the following output:
+
+      .. code-block:: none
+
+         fw  fwupx64.efi  grub.cfg  grubx64.efi  MokManager.efi  shimx64.efi
+
+   #. Install the hypervisor (``acrn.efi``)
+
+      .. code-block:: console
+
+         sudo mkdir /mnt/EFI/acrn/
+         sudo cp ~/acrn-hypervisor/build/hypervisor/acrn.efi /mnt/EFI/acrn
+
+   #. Configure the EFI firmware to boot the ACRN hypervisor by default
+
+      .. code-block:: console
+
+         sudo efibootmgr -c -l "\EFI\acrn\acrn.efi" -d /dev/sda -p 1 -L "ACRN Hypervisor" -u "bootloader=\EFI\ubuntu\grubx64.efi"
+         sudo efibootmgr -v # to verify that the "ACRN Hypervisor" will be booted first
+
+      You can change the boot order at any time using ``efibootmgr -o XXX,XXX,XXX``
+
+      .. note::
+         By default, the “ACRN Hypervisor” you have just added should be the first one to boot but it is a recommended
+         that you double-check that using ``efibootmgr -v`` or by entering the EFI firmware (``F10``)
+
+Install the Service OS kernel
+*****************************
+
+You can download latest Service OS kernel from `<https://download.clearlinux.org/releases/current/clear/x86_64/os/Packages/>`_
+(**We need to update the URL to one that is known to work, and matches the tag/release we use above!!!**). We will extract it
+and install it in the next few steps.
+
+a. Download and extract the Service OS kernel
+
+   .. code-block:: console
+
+      mkdir ~/kernel-build
+      cd ~/kernel-build
+      wget  https://download.clearlinux.org/releases/current/clear/x86_64/os/Packages/linux-pk414-sos-4.14.41-39.x86_64.rpm
+      sudo apt-get install rpm2cpio
+      rpm2cpio linux-pk414-sos-4.14.41-39.x86_64.rpm | cpio -idmv
+
+#. Install the SOS kernel and its drivers (modules)
+
+   .. code-block:: console
+
+      sudo cp -r ~/kernel-build/usr/lib/modules/4.14.41-39.pk414-sos/ /lib/modules/
+      sudo cp ~/kernel-build/usr/lib/kernel/org.clearlinux.pk414-sos.4.14.41-39 /boot/acrn/
+
+#. Configure Grub to load the Service OS kernel
+
+   * Modify the ``/etc/grub.d/40_custom`` file to create a new Grub entry that will boot the SOS kernel.
+
+     .. code-block:: none
+
+        menuentry 'ACRN ubuntu SOS' {
+                recordfail
+                load_video
+                insmod gzio
+                insmod part_gpt
+                insmod ext4
+                linux /boot/acrn/org.clearlinux.pk414-sos.4.14.41-39 pci_devices_ignore=(0:18:1) maxcpus=1 console=tty0 console=ttyS0
+                i915.nuclear_pageflip=1 root=PARTUUID=<ID of rootfs partition> rw rootwait ignore_loglevel no_timer_check consoleblank=0
+                i915.tsd_init=7 i915.tsd_delay=2000 i915.avail_planes_per_pipe=0x00000F i915.domain_plane_owners=0x011111110000
+                i915.enable_guc_loading=0 i915.enable_guc_submission=0 i915.enable_preemption=1 i915.context_priority_mode=2
+                i915.enable_gvt=1 hvlog=2M@0x1FE00000
```

Comment:
Review all parameters to make sure they match the version of ACRN/Service OS kernel we use